src/commands/Suggestion.js

```diff
@@ -88,6 +86,33 @@ const _ProcessSuggestion = async (msg, guild, locale, suggestionId, reason, Sugg
         ));
     }
 
+    switch (SuggestionStatus) {
+        case 1:
+            embed.setColor(Number.parseInt(locale.Get("suggestionApprovedColor")));
+            embed.setTitle(locale.GetFormatted("suggestionApprovedTitle", { "id": suggestionId }));
+            embed.setDescription(locale.GetFormatted(
+                "suggestionApprovedDescription",
+                { "user-mention": Utils.MentionUser(suggestionRow.authorId) }
+            ));
+            break;
+        case 2:
+            embed.setColor(Number.parseInt(locale.Get("suggestionRejectedColor")));
+            embed.setTitle(locale.GetFormatted("suggestionRejectedTitle", { "id": suggestionId }));
+            embed.setDescription(locale.GetFormatted(
+                "suggestionRejectedDescription",
+                { "user-mention": Utils.MentionUser(suggestionRow.authorId) }
+            ));
+            break;
+        case 3:
+            embed.setColor(Number.parseInt(locale.Get("suggestionImplementedColor")));
+            embed.setTitle(locale.GetFormatted("suggestionImplementedTitle", { "id": suggestionId }));
+            embed.setDescription(locale.GetFormatted(
+                "suggestionImplementedDescription",
+                { "user-mention": Utils.MentionUser(suggestionRow.authorId) }
+            ));
+            break;
+    }
```

Comment:
Switch has no `default` branch. I would suggest having `SuggestionStatus.Approved` be the default behaviour, or log the invalid state.